rounding.scad

```diff
@@ -1990,13 +1990,13 @@ function bezier_patch_degenerate(patch, splinesteps=16, reverse=false) =
     // at this point top_degen is true                   // only top is degenerate
        let(
            full_degen = patch[1][0] == last(patch[1]),
-           rowmax = full_degen ? range(splinesteps+1) :
+           rowmax = full_degen ? count(splinesteps+1) :
                                  [for(j=[0:splinesteps]) j<=splinesteps/2 ? 2*j : splinesteps],
            vbb=echo("single degenerate case"),
            bpatch = [for(i=[0:1:len(patch[0])-1]) bezier_points(subindex(patch,i), samplepts)],
            pts = [
                   [bpatch[0][0]],
-                  for(j=[1:splinesteps]) bezier_points(subindex(bpatch,j), range(rowmax[j]+1)/rowmax[j])
+                  for(j=[1:splinesteps]) bezier_points(subindex(bpatch,j), count(rowmax[j]+1)/rowmax[j])
```

Comment:
Shouldn't this be for(..) bezier_points(subindex(bpatch,j), lerpn(0,1,rowmax[j]+1)